dsgrid/config/project_config.py

```diff
@@ -774,6 +774,21 @@ def has_dataset(self, dataset_id, status=None):
         # TODO: what about benchmark and historical?
         return False
 
+    def get_load_data_time_columns(self, dimension_query_name):
+        """Return the time dimension columns expected in the load data table for this query name.
+
+        Parameters
+        ----------
+        dimension_query_name : str
+
+        Returns
+        -------
+        list
+        """
+        dim = self.get_dimension(dimension_query_name)
+        time_columns = dim.get_timestamp_load_data_columns()
```

Comment:
Should this be `dim.get_time_load_data_columns()`? Or just mimic the name here, `get_load_data_time_columns`?

Reply:
This was named long ago. I can change it if it won't cause merge conflicts for @lixiliu